### PR TITLE
Add pregnancy questionnaire to patient careplans missing it.

### PR DESCRIPTION
**Pull request by pbugni**
Fix for https://www.pivotaltracker.com/story/show/172413182

### CODE REVIEW

File: map/migrations/versions/add_prego_questionnaire.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+        cp, changed = add_missing_questionnaire(cp)
+        if changed:
+            print(
+                "Added missing Questionnaire to CarePlan %s for "
+                "%s" % (cp['id'], str(cp['subject'])))
+            result, status = HapiRequest.put_resource(cp)
+            if status != 200:
+                print(
+                    "Failed with status %d: %s" % (status, result.text))
```
Comment (hannahburkhardt) [inline map/migrations/versions/add_prego_questionnaire.py:57]: Looks like this part is duplicated between this migration and map-api/map/migrations/versions/extend_careplan.py . Would it be appropriate to consolidate that?

Comment (pbugni) [reply map/migrations/versions/add_prego_questionnaire.py:57]: well yes, i was just following the migration model.  i'll work on putting forth a utility in the name of DRY

Comment (pbugni) [reply map/migrations/versions/add_prego_questionnaire.py]: @hannahburkhardt ready for another look

Comment (hannahburkhardt) [reply map/migrations/versions/add_prego_questionnaire.py]: Most excellent!

**Pull request merged**